server/src/main/java/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -2006,9 +2007,9 @@
         sb.and("display", sb.entity().isDisplayVolume(), SearchCriteria.Op.EQ);
         sb.and("state", sb.entity().getState(), SearchCriteria.Op.EQ);
         sb.and("stateNEQ", sb.entity().getState(), SearchCriteria.Op.NEQ);
-        sb.and("systemUse", sb.entity().isSystemUse(), SearchCriteria.Op.NEQ);
+        sb.and("systemUse", sb.entity().isSystemUse(), SearchCriteria.Op.IN);
```

Comment:
Then we won't need this change

Reply:
@nvazquez I still need this change since I need to assign both 0 and 1 for root admin